src/components/inspectors/CycleManager.js

```diff
@@ -5,17 +5,17 @@ export default class CycleManager {
   #startDate;
   #repeat;
   #periodicityValue;
-  #selectedWeekdays;
-  #endDate;
+  _selectedWeekdays;
+  _endDate;
```

Comment:
@lisa-fehr Were these two properties causing tests to fail? I think having some properties use the underscore and others the hash makes it a little harder for me to read compared to having them all follow the same convention.

If there's issues with the private class property syntax then sticking with underscores is fine.

Reply:
Yeah. It is a bit strange. I can't see a direct alteration of the private value.